packages/commonwealth/client/scripts/views/components/AuthButton/AuthButton.scss

```diff
@@ -5,29 +5,63 @@
   align-items: center;
   gap: 16px;
   border-radius: 6px;
-  border: 2px solid $neutral-200;
   padding: 16px;
   width: 100%;
   outline: none !important;
   cursor: pointer;
-  background-color: white;
-
-  &:not(:disabled) {
-    &:focus-within,
-    &:focus,
-    &:active {
-      border: 2px solid $primary-200 !important;
-      outline: none !important;
-    }
+
+  &.rounded {
+    border-radius: 16px;
   }
 
-  &:disabled {
-    background-color: $neutral-100 !important;
-    cursor: not-allowed;
+  &.light {
+    background-color: white;
+    border: 2px solid $neutral-200;
+
+    &:not(:disabled) {
+      &:focus-within,
+      &:focus,
+      &:active {
+        border: 2px solid $primary-200 !important;
+        outline: none !important;
+      }
+    }
+
+    &:disabled {
+      background-color: $neutral-100 !important;
+      cursor: not-allowed;
+    }
+
+    &:hover {
+      background-color: $neutral-25;
+    }
   }
 
-  &:hover {
-    background-color: $neutral-25;
+  &.dark {
+    background-color: #1e1f1f;
```

Comment:
can you use color from pallete

Reply:
this was a 1-off color usage, not sure if we want to add it to global color, but if yes, plz lmk.